lib/galaxy/tool_util/parser/xml.py

```diff
@@ -914,9 +914,9 @@ def parse_stdio_exit_codes(self, stdio_elem):
                 exit_code.error_level = (
                     self.parse_error_level(exit_code_elem.get("level")))
                 code_range = exit_code_elem.get("range", "")
-                if code_range is None:
+                if code_range == "":
```

Comment:
Any reason not to drop the default `""` one live above instead and in the other cases below ?

Reply:
No, I also thought about that.